### ISSUE TITLE
How to save From field after change

### ISSUE THREAD

**Issue by exander77** [mjl-/mox#245]
I can't find a way to change From field from UI permanently, only from conf file. I think there should be a save button in UI where after editing from you can save it.

**Comment by mjl-** [comment]
I think this is about the webmail and the "From" address of the "compose message" popup?
You have multiple addresses configured, and the wrong address gets picked by default?
For a reply, the webmail should pick the address that was in the original "To" message header.
For new messages, the webmail picks the email address used to log in. You can login with any address that's configured for an account, all with the same password.

Or perhaps you're thinking about another From field?

**Comment by exander77** [comment]
No, my question is more mundane, how to set FullName configuration option from Webmail?

**Comment by mjl-** [comment]
The full name can only be configured from the account page. You can configure a full name for each address in the account. I didn't add it to the settings of the webmail because then we would have two places to edit the same thing...
Perhaps it would help to add a link to the account page in the settings popup? It could have a line saying something like "For more settings see the account page.".

I have thought about completely integrating the account page with the webmail page. But some admins may want to enable/disable the webmail and account pages separately. For example, don't offer a webmail interface. Then without webmail interface, you couldn't change many account settings...
